### PR TITLE
aws-17-1-0

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/roadmap/issues/114

### CODE REVIEW

**Comment by whites11** [comment]
Please wait for next version of net-exporter

File: aws/v17.1.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    version: 1.22.2-gs3
+  - componentVersion: 1.8.6
+    name: coredns
+    version: 1.8.0
+  - componentVersion: 0.10.2
+    name: external-dns
+    version: 2.9.0
+  - componentVersion: 4.2.0
+    name: kiam
+    version: 2.1.0
```
Comment (AverageMarcus) [inline aws/v17.1.0/release.yaml:32]: What are we going to do about Kiam?

Comment (calvix) [reply aws/v17.1.0/release.yaml:32]: good point, forgot about it, can we just bump the version? is that changed kiam working for clusters with and without IRSA?

Comment (AverageMarcus) [reply aws/v17.1.0/release.yaml:32]: No, it's a variant that defaults the whitelist value. I didn't realise Kiam was part of the release, I'm now not sure if the variant is the correct approach.

I'm wondering if we leave Kiam as-is and include in the IRSA docs that the user-values need providing (like we did in the test)

Comment (calvix) [reply aws/v17.1.0/release.yaml:32]: lets talk in planning

**Comment by calvix** [comment]
Release is ready from my side, handing over to @alex-dabija

**Comment by calvix** [comment]
@alex-dabija this is the commit for the new permissions that needed to be added to aws-operator ( done by Nick)
https://github.com/giantswarm/giantswarm-aws-account-prerequisites/commit/35a1bdc67c7035b74e6200a4ff644a1ad00f3fbb

**Review by alex-dabija — APPROVED** [review]

**Pull request merged**